integration/obscurogateway/gateway_user.go

```diff
@@ -1,4 +1,4 @@
-package faucet
+package obscurogateway
```

Comment:
obscurogateway -> tengateway for the whole package

Reply:
I just aligned it with the folder name. Let's change this as part of a smaller pr